src/EpisodeManagementService/GetEpisode/GetEpisode.cs

```diff
@@ -0,0 +1,61 @@
+using System.Net;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Azure.Functions.Worker.Http;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Configuration;
+using NHS.ServiceInsights.Common;
+
+namespace NHS.ServiceInsights.EpisodeManagementService
+{
+    public class GetEpisode
+    {
+        private readonly ILogger<GetEpisode> _logger;
+        private readonly IHttpRequestService _httpRequestService;
+        private readonly IConfiguration _configuration;
+
+        public GetEpisode(ILogger<GetEpisode> logger, IHttpRequestService httpRequestService, IConfiguration configuration)
```

Comment:
We are no longer using IConfiguration so it can be removed

Reply:
IConfiguration removed.